drivers/console/rtt_console.c

```diff
@@ -41,9 +41,7 @@ static int rtt_console_out(int character)
 	int max_cnt = CONFIG_RTT_TX_RETRY_CNT;
 
 	do {
-		SEGGER_RTT_LOCK();
-		cnt = SEGGER_RTT_WriteNoLock(0, &c, 1);
-		SEGGER_RTT_UNLOCK();
+		cnt = SEGGER_RTT_Write(0, &c, 1);
```

Comment:
For another PR but could we make the channel configurable?

Reply:
Even simple, I agree to do it in another PR :-)